app/jobs/dradis/plugins/csv/mapping_import_job.rb

```diff
@@ -0,0 +1,82 @@
+require 'csv'
+
+module Dradis::Plugins::CSV
+  class MappingImportJob < ApplicationJob
+    queue_as :dradis_project
+
+    def perform(file:, identifier_col_index:, mappings:, project_id:, uid:)
+      @logger = Log.new(uid: uid)
+      @logger.write { "Job id is #{job_id}." }
+      @logger.write { 'Worker process starting background task.' }
+
+      @file = file
+      @identifier_col_index = identifier_col_index
+      @mappings = mappings
+      @project = Project.find(project_id)
+      node_col_mapping = @mappings.find { |index, field| field['type'] == 'node' }
+      @node_col_index = node_col_mapping[0].to_i if node_col_mapping
```

Comment:
Why not just call `first` after the the `find` instead of doing this in 2 steps?

Reply:
Good idea, thanks

Reply:
Actually, this doesn't work because `nil.to_i` evaluates to `0` and it'll use the first column as the node label.

Would this be better?
```ruby
      @node_index = 
        if node_mapping = @mappings.find { |index, field| field['type'] == 'node' }&.first
          node_mapping[0].to_i
        end
```